packaging/userspace/userspace-packager.py

```diff
@@ -127,16 +130,26 @@ def __init__(
 		super().__init__(context, executor)
 
 		if not self.options.target_platform:
-			distro = linux_distribution(full_distribution_name=False)
-			self.options.target_platform = distro[0].lower()
-			self.options.target_platform_variant = distro[1].partition('.')[0].lower()
+			plat = distro.linux_distribution(full_distribution_name=False)
+			self.options.target_platform = plat[0].lower()
+			self.options.target_platform_variant = plat[1].partition('.')[0].lower()
+		# This will likely change once we are building on ubi8
```

Comment:
what does this mean?  ub18...

Reply:
red hat universal base image. It's what rocky, alma, oracle, amazon, centos, et al are built on top of.

Reply:
oh, ubi, i totally read it as ubuntu18.   carry on.